.env.example

```diff
@@ -4,6 +4,12 @@
 # Database URL (default : the "mobilic" DB on your local psql server)
 DATABASE_URL=
 
+#S3 (scaleway when introduced)
```

Comment:
#suggestion(non-blocking): if you think we may change in the future, you can rename the variables SCW_* by S3_*